### DISCUSSION TITLE
The default build configuration of Vue only support most newest version of browser, Please compile a version for legacy browser.

### DISCUSSION THREAD

**Discussion by ghost** [cdnjs/cdnjs#14207] [General]
Add a configuration then modify babel build target to IE11

**Comment by MattIPv4** [comment]
cdnjs does not build Vue. We distribute the prebuilt version in the Vue NPM package.

If that version does not support the browsed you need, you will need to take this up with the Vue maintainers so they can ship a new version that has better compatibility.
